src/anyio/__init__.py

```diff
@@ -1,24 +1,26 @@
-           'ExceptionGroup', 'IncompleteRead', 'TypedAttributeLookupError', 'WouldBlock',
-           'AsyncFile', 'open_file', 'aclose_forcefully', 'open_signal_receiver', 'connect_tcp',
-           'connect_unix', 'create_tcp_listener', 'create_unix_listener', 'create_udp_socket',
-           'create_connected_udp_socket', 'getaddrinfo', 'getnameinfo', 'wait_socket_readable',
-           'wait_socket_writable', 'create_memory_object_stream', 'run_process', 'open_process',
-           'create_lock', 'CapacityLimiter', 'CapacityLimiterStatistics', 'Condition',
-           'ConditionStatistics', 'Event', 'EventStatistics', 'Lock', 'LockStatistics',
-           'Semaphore', 'SemaphoreStatistics', 'create_condition', 'create_event',
-           'create_semaphore', 'create_capacity_limiter', 'open_cancel_scope', 'fail_after',
-           'move_on_after', 'current_effective_deadline', 'TASK_STATUS_IGNORED', 'CancelScope',
-           'create_task_group', 'TaskInfo', 'get_current_task', 'get_running_tasks',
-           'wait_all_tasks_blocked', 'run_sync_in_worker_thread', 'run_async_from_thread',
-           'run_sync_from_thread', 'current_default_worker_thread_limiter',
-           'create_blocking_portal', 'start_blocking_portal', 'typed_attribute',
-           'TypedAttributeSet', 'TypedAttributeProvider')
+__all__ = ('maybe_async', 'maybe_async_cm', 'run', 'sleep', 'sleep_until', 'current_time',
+           'get_all_backends', 'get_cancelled_exc_class', 'BrokenResourceError',
+           'BrokenWorkerProcess', 'BusyResourceError', 'ClosedResourceError', 'DelimiterNotFound',
+           'EndOfStream', 'ExceptionGroup', 'IncompleteRead', 'TypedAttributeLookupError',
+           'WouldBlock', 'AsyncFile', 'open_file', 'aclose_forcefully', 'open_signal_receiver',
+           'connect_tcp', 'connect_unix', 'create_tcp_listener', 'create_unix_listener',
+           'create_udp_socket', 'create_connected_udp_socket', 'getaddrinfo', 'getnameinfo',
+           'wait_socket_readable', 'wait_socket_writable', 'create_memory_object_stream',
+           'run_process', 'open_process', 'create_lock', 'CapacityLimiter',
+           'CapacityLimiterStatistics', 'Condition', 'ConditionStatistics', 'Event',
+           'EventStatistics', 'Lock', 'LockStatistics', 'Semaphore', 'SemaphoreStatistics',
+           'create_condition', 'create_event', 'create_semaphore', 'create_capacity_limiter',
+           'open_cancel_scope', 'fail_after', 'move_on_after', 'current_effective_deadline',
+           'TASK_STATUS_IGNORED', 'CancelScope', 'create_task_group', 'TaskInfo',
+           'get_current_task', 'get_running_tasks', 'wait_all_tasks_blocked',
+           'run_sync_in_worker_thread', 'run_async_from_thread', 'run_sync_from_thread',
+           'current_default_worker_thread_limiter', 'create_blocking_portal',
+           'start_blocking_portal', 'typed_attribute', 'TypedAttributeSet',
```

Comment:
perhaps one entry per line is better to minimize changed lines and conflicts

Reply:
I've made this change in master. Thanks for the suggestion.